src/input/Input.tsx

```diff
@@ -130,15 +130,17 @@ export const InputField = forwardRef<
             {...rest}
           />
         )}
-
-        <label htmlFor={id}>
-          {type === "checkbox" && (
-            <div className={styles.checkbox}>
-              <CheckIcon />
-            </div>
-          )}
-          {label}
-        </label>
+        {/* Dont create label element if not set -> prohibits broken animation on focus*/}
```

Comment:
For screen reader accessibility and usability in general, it's important that we always label fields. Even though the designs technically don't show a label on the feedback field (only a placeholder), I suggest we use 'Your feedback' as the label. (It could easily have been an accidental omission in the designs)

Reply:
this makes this option to not add a label very bad. reverted it and changed to look to diverge from the designs and showing the label in the feedback text field